_posts/2017/2017-02-20-continuous-documentation-with-ansible.md

```diff
@@ -0,0 +1,207 @@
+---
+layout: post
+title: "Continuous Documentation with Ansible"
+date: "2017-02-20 06:39:20"
+image: blog-header/heartbeat.jpg
+categories: tech-stories agile devops docker jenkins ansible
+authors: ["Thomas H."]
+---
+
+# Introduction
```

Comment:
don't use header1 ('#') in your text since jekyll uses it already for the blog title (http://www.jekyllnow.com/Markdown-Style-Guide/)